### PR TITLE
Expose verifyDigest

### PR DESCRIPTION
**Pull request by cf19drofxots**

### CODE REVIEW

File: src.ts/utils/index.ts [hunk 1]
```diff
@@ -92,6 +92,7 @@ export {
 
     computePublicKey,
     computeSharedSecret,
-    verifyMessage
+    verifyMessage,
+    verifyDigest
```
Comment (armaniferrante) [inline src.ts/utils/index.ts]: Why do we need this?

Comment (cf19drofxots) [reply src.ts/utils/index.ts]: The SignerKey interface exposes a method `signDigest` which takes in a digest and returns a signature.

`verifyMessage` on the other hand accepts a (message, signature) pair. It then hashes the message, then passes this hash to `verifyDigest`.

I want the functionality of `verifyDigest` as I already have the digest, and I don't want it being hashed again after passing it in.

Comment (armaniferrante) [reply src.ts/utils/index.ts]: Can you link me to the place this will be used?

Comment (cf19drofxots) [reply src.ts/utils/index.ts]: No specific place yet, just playing around.

Something like this basically:
```
> let ethers = require("ethers")
undefined
> const message = "0x111111";
undefined
> const signer = new ethers.SigningKey("0xf2f48ee19680706196e2e339e5da3491186e0c4c5030670656b0e0164837257d")
undefined
> const signature = signer.signDigest(message);
undefined
> signature
{ recoveryParam: 1,
  r: '0x79d47be12761332475668f65489ae223b106ce015672fed60a24712bf4a3e5cb',
  s: '0x79a54b7ae890a86eb1a7858680d51a9f5197dc08fedcadb145117296e6d301a6',
  v: 28 }
> signer.address
'0x5409ED021D9299bf6814279A6A1411A7e866A631'
> ethers.utils.verifyMessage(message, signature);
'0xD64583e5B5dDB2f35996353EfAc9B1980dbEe1b5'
> ethers.utils.verifyDigest(message, signature);
TypeError: ethers.utils.verifyDigest is not a function
```

Comment (cf19drofxots) [reply src.ts/utils/index.ts]: @armaniferrante with this change, the new functionality that we get is this:

```
> let ethers = require("ethers")
undefined
> const message = "0x111111";
undefined
> const signer = new ethers.SigningKey("0xf2f48ee19680706196e2e339e5da3491186e0c4c5030670656b0e0164837257d")
undefined
> const signature = signer.signDigest(message);
undefined
> signature
{ recoveryParam: 1,
  r: '0x79d47be12761332475668f65489ae223b106ce015672fed60a24712bf4a3e5cb',
  s: '0x79a54b7ae890a86eb1a7858680d51a9f5197dc08fedcadb145117296e6d301a6',
  v: 28 }
> signer.address
'0x5409ED021D9299bf6814279A6A1411A7e866A631'
> ethers.utils.verifyDigest(message, signature);
'0x5409ED021D9299bf6814279A6A1411A7e866A631'
> signer.address === ethers.utils.verifyDigest(message, signature);
true
```

**Pull request merged**